classes/wp-background-process.php

```diff
@@ -62,13 +62,9 @@ public function __construct() {
 		add_filter( 'cron_schedules', array( $this, 'schedule_cron_healthcheck' ) );
 	}
 
-	/**
-	 * Dispatch
-	 *
-	 * @access public
-	 * @return void
-	 */
-	public function dispatch() {
+
+	public function dispatch(): array
```

Comment:
```suggestion
	final public function dispatch(): array
```

Reply:
The existing classes have to be reworked before this can happen.